<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMni</class>
 <widget class="QDialog" name="DialogMni">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление МНИ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>МНИ</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="cbRegWho"/>
            </item>
            <item>
             <widget class="QPushButton" name="pbAddUser">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="flash-admin.qrc">
                <normaloff>:/icons/add_user</normaloff>:/icons/add_user</iconset>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leRegNumber"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leTitle"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leSerial"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leManufacturer"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="leProduct"/>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="leVID"/>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lePID"/>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="leComment"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>На кого зарегистрован:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Уч. номер</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Примечание</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>PID:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Наименование</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Серийный номер</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Производитель:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Продукт:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>VID:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Разрешено использование на</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chbPersonSVT">
          <property name="text">
           <string>СВТ ответственного лица</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbDivisionSvt">
          <property name="text">
           <string>всех СВТ подразделения ответственного лица</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbAllSvt">
          <property name="text">
           <string>Всех СВТ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbAddSVT">
          <property name="text">
           <string>Добавить СВТ</string>
          </property>
          <property name="icon">
           <iconset resource="flash-admin.qrc">
            <normaloff>:/icons/add_svt</normaloff>:/icons/add_svt</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actAutoLoadMNI">
   <property name="icon">
    <iconset resource="flash-admin.qrc">
     <normaloff>:/icons/autoUSB</normaloff>:/icons/autoUSB</iconset>
   </property>
   <property name="text">
    <string>Автоматически загрузить данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="flash-admin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMni</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbPersonSVT</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DialogMni</receiver>
   <slot>chbChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbDivisionSvt</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DialogMni</receiver>
   <slot>chbChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbAllSvt</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DialogMni</receiver>
   <slot>chbChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chbChanged()</slot>
 </slots>
</ui>
